src/gui/widgets/scroll_text.hpp

```diff
@@ -163,6 +165,12 @@ class scroll_text : public scrollbar_container
 
 	void place(const point& origin, const point& size);
```

Comment:
Getting a compiler warning with Clang that `place` could be declared `override`, but isn't.

It's unrelated to this PR, and it's a trivial warning (doesn't change the compilation result), so I suggest remembering it and doing it in a follow-up PR.